openflow_input/bsn_tlv

```diff
@@ -963,3 +963,16 @@ struct of_bsn_tlv_data_mask : of_bsn_tlv {
     uint16_t length;
     of_octets_t value;
 };
+
+enum ofp_bsn_port_usage(wire_type=uint16_t) {
```

Comment:
Is this a OF config message, requesting the interface to only be used in a specific direction?